tools/DependencyUpdater/Program.cs

```diff
@@ -884,42 +899,33 @@ private static void UpdateProject(string projectToUpdate, string repositoryPath,
             string libraryName,
             string commitMessage,
             string newBranchName,
-            string prTitle)
+            string prTitle,
+            string repoOwner)
         {
             try
             {
                 // create PR
```

Comment:
Missing the special processing of AMQPLiteLibraryName.... without it, PRs against that repo can't be automated

Reply:
Previously, you were using
`                 // AMQPLite
                if (libraryName == AMQPLiteLibraryName)
                {
                    repoOwner = "Azure";
                }`

Now repoOwner is extracted from repository link and passed into this method. So my thought were that it should work that way so I deleted this lines. 
But if it won't work, I think better idea is to include "repoOwner" in cmd parameters.

Reply:
Good point! And yes, this can (should) be passed as a parameter.